docs/sources/tutorials/send-logs-to-loki.md

```diff
@@ -1,26 +1,28 @@
 ---
-canonical: https://grafana.com/docs/alloy/latest/tutorials/get-started/
-description: Getting started with Alloy
-title: Get started with Alloy
-weight: 10
+canonical: https://grafana.com/docs/alloy/latest/tutorials/send-logs-to-loki/
+aliases:
+  - ./get-started/ #/docs/alloy/latest/tutorials/get-started/
+description: Learn how to use Grafana Alloy to send logs to Loki
+menuTitle: Send logs to Loki
+title: Use Grafana Alloy to send logs to Loki
+weight: 100
 ---
 
-## Get started with {{% param "PRODUCT_NAME" %}}
+## Use {{% param "FULL_PRODUCT_NAME" %}} to send logs to Loki
 
-This tutorial shows you how to configure {{< param "PRODUCT_NAME" >}} to collect logs from your local machine, filter non-essential log lines, and send them to Loki, running in a local Grafana stack.
+This tutorial shows you how to configure {{< param "PRODUCT_NAME" >}} to collect logs from your local computer, filter non-essential log lines, send them to Loki, and use a Grafana dashboard to query and visualize the results.
```

Comment:
I would leave the term local machine here rather than changing it to a local computer. 

Our readers have diverse setups which may include computers, work stations, servers, mobile devices, and development boards. 

The term local machine captures all of these.